javascript/index.d.ts

```diff
@@ -43,6 +43,7 @@ declare class Session {
     create_outbound(
         account: Account, their_identity_key: string, their_one_time_key: string,
     ): void;
+    create_outbound_without_otk(account: Account, their_identity_key: string): void;
```

Comment:
The TypeScript types here are out-of-date and don't include most of the params. I think we should either update it to be correct, or ignore it completely

Reply:
I'll ignore it, but going to add proper definition to Flow in `comm` codebase